theseus/src/api/profile_create.rs

```diff
@@ -74,13 +74,12 @@ pub async fn profile_create(
         None
     };
 
-    // Fully canonicalize now that its created for storing purposes
-    let path = canonicalize(&path)?;
-    let mut profile =
-        Profile::new(uuid, name, game_version, path.clone()).await?;
+    let mut profile = Profile::new(uuid, name, game_version).await?;
```

Comment:
shouldn't we check if the name is unique here? maybe we can append like Profile (1) etc if it isn't unique?